src/components/ui/SuggestItem.jsx

```diff
@@ -16,6 +17,18 @@ const GeolocationItem = () =>
     <ItemLabels firstLabel={_('Your position', 'direction')} />
   </div>;
 
+const IntentionItem = ({ intention }) => {
+  const { category, place } = intention;
+  const placeString = place
+    ? `${_('Close to')} ${place.properties.geocoding.name}`
+    : _('Search around this place');
+
+  return <div className="autocomplete_suggestion autocomplete_suggestion--intention">
+    <div className="autocomplete-icon" />
+    <ItemLabels firstLabel={category.name} secondLabel={placeString} />
```

Comment:
The category `name` is not localized here.
```suggestion
    <ItemLabels firstLabel={category.label} secondLabel={placeString} />
```

Reply:
Damn it, once again! :roll_eyes: Thanks, I pushed your fix.

Reply:
Damn it, once again! :roll_eyes: Thanks, I pushed your fix.